packages/nimble-components/src/wafer-map/specs/features/padding.md

```diff
@@ -0,0 +1,38 @@
+# Nimble Wafer Map Die Padding
```

Comment:
Make sure to link to this spec from the top-level wafer-map README.md spec
We may want to add a features section to that document to make it easier to reference the related feature specs